integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -752,6 +746,27 @@ public void testStandaloneManagedRestartIfNeeded() {
     assertTrue(result.contains("No changes needed"), "startServer.sh shouldn't make changes");
   }
 
+  /**
+   * Start a dynamic cluster managed server within the max cluster size
+   * and verify the replica count increased.
```

Comment:
Here we do not check the repilca count. Here we can check the stdout of the startupScript which indicate that replica count has been increased OR we can create a method to get the updated replica count from the domain resource.

Reply:
changed